circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  node:
+    version: 6.9.2
```

Comment:
We need to run tests for the various node versions for non-server pieces - the client needs to run on 4-7